config/locales/assessor_interface.en.yml

```diff
@@ -145,19 +152,21 @@ en:
           attributes:
             assessor_id:
               blank: Select an assessor to assign
+        assessor_interface/assessment_recommendation_form:
+          attributes:
+            recommendation:
+              blank: Select your recommendation
+              inclusion: Select your recommendation
+            declaration:
+              blank: You must confirm that you have reviewed this QTS application
+            confirmation:
+              inclusion: Select whether you want to confirm this application
```

Comment:
This doesn't make sense when it's a decline that we're confirming.

Reply:
Good point, after discussing with Wes I've changed this so the message is dynamic depending on whether the user is awarding or declining.